src/v3/test/integration/__snapshots__/authenticator-enroll-security-question.test.tsx.snap

```diff
@@ -296,13 +296,19 @@ exports[`authenticator-enroll-security-question custom question renders correct
                           </fieldset>
                         </div>
                       </div>
+                      <p
+                        class="MuiFormHelperText-root Mui-error emotion-50"
+                        data-se="credentials.answer-error"
+                      >
+                        This field cannot be left blank
+                      </p>
```

Comment:
This should go away once you include all field types (as opposed to only targeting input fields)